plugins/services/src/js/constants/OperatorTypes.js

```diff
@@ -1,13 +1,11 @@
-const DEFAULT_HELP = "A string, integer or regex value.";
-const OPTIONAL_HELP = `${DEFAULT_HELP} This field is optional.`;
+const OPTIONAL_HELP = "optional";
```

Comment:
I think this should be "This field is optional" - my fault just typing "optional" when doing the mockup 👍

Reply:
Also, after your change, this is basically only a true/false flag, right? It should be possible to rewrite it that it uses `requiresValue` - which means we should be able to remove it from the file 🕺

Reply:
You are right, let's make it a bool 👍